resources/js/components/templates/TemplateSearch.vue

```diff
@@ -72,7 +73,16 @@
         size="sm"
         last-number
         first-number
-      ></b-pagination>
+        ></b-pagination>
+        <div>
+          <a href="https://www.processmaker.com/resources/customer-success/templates/" 
+            class="text-muted"
+            target="_blank">
+            Visit our Gallery for more Templates
```

Comment:
I think this should have its translation too

Reply:
Good catch! Let me fix it